### ISSUE TITLE
Separate server and client components

### ISSUE THREAD

**Issue by sfrdmn** [odysseyscience/react-s3-uploader#41]
Seems like it would be more versatile to separate the server and client components into separate (NPM) modules. People should be able to install just the React bit without then grabbing the Express dependency

**Comment by seanadkinson** [comment]
Yeah, I completely agree.  I've wanted to do this for awhile, but just never got around to it.  I think in the short term, it would be really easy to just specify `express` as an `optionalDependency`, and then people who don't want the bundled server don't need to include it.

**Comment by clarkie** [comment]
@sfrdmn as of #53 (v2.0.0) express is no longer pulled as part of `npm install`.
